__tests__/app.test.js

```diff
@@ -117,3 +118,62 @@ describe("/api/articles", () => {
          });
    });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+   test("GET 200: Should respond with an array of all comments for the article of a specific article_id", () => {
+      return request(app)
+         .get("/api/articles/5/comments")
+         .expect(200)
+         .then(({ body: { comments } }) => {
+            expect(comments.length).toBeGreaterThan(0);
```

Comment:
You can be more specific here - how many comments do you expect this article to have?